config/locales/pt-BR.yml

```diff
@@ -54,6 +54,7 @@ pt-BR:
   remove: "Remover"
   add user: "Adicionar usuário"
   add new member: "Adicionar um novo membro"
+  add new label: "Adicionar uma nova etiqueta"
```

Comment:
`Etiqueta` sounds weird to me. Why not `tag`?

Reply:
For me too, but we have this: `labels:  'Etiquetas'` in pt-BR translation, so I think we should change all at once in other PR.

Reply:
LGTM 👍